packages/vscode-extension/src/debug/depsChecker/vscodeLogger.ts

```diff
@@ -56,7 +56,7 @@ export class VSCodeLogger implements DepsLogger {
   }
 
   private addToDetailCache(level: LogLevel, message: string): void {
-    const line = `${LogLevel[level]} ${new Date().toISOString()}: ${message}`;
+    const line = `${String(LogLevel[level])} ${new Date().toISOString()}: ${message}`;
```

Comment:
Keep consistency? `String()` or `as string` for all changes

Reply:
Now I am using `String()` to convert other types (such as numbers, booleans, and undefined values) into strings and using the type assertion `as string` to convert a variable of type any to a string.